pkg/cmd/cli/cmd/process.go

```diff
@@ -149,8 +168,8 @@ func RunProcess(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []
 		obj, ok := infos[i].Object.(*api.Template)
 		if !ok {
 			sourceName := filename
-			if len(storedTemplate) > 0 {
-				sourceName = namespace + "/" + storedTemplate
+			if len(templateName) > 0 {
+				sourceName = namespace + "/" + templateName
```

Comment:
This also brings me to another problem, what happen when I do:

```
oc process foo-template openshift//another-template
```

In this case, what I can see now is that the first stored template is processed and returned to console and the second one is ignored. I don't think this PR caused this and I think it is rather a bug.